src/models/base.py

```diff
@@ -174,6 +193,32 @@ def evaluate(self, save_results: bool = True, save_preds: bool = False) -> None:
                     .to_xarray()
                 )
 
+                if self.predict_delta:
```

Comment:
is the predict_delta stuff in here too??

Reply:
i must have pulled from the predict_delta branch (was meant to go the other way because i needed these functions)

Reply:
Okay - let's merge the predict_delta branch before merging this one